### DISCUSSION TITLE
It will be nice for me if any of you get involved in this story | Мне будет приятно, если кто-нибудь из вас примет участие в этой истории.

### DISCUSSION THREAD

**Discussion by georgetoloraia** [community/community#102628] [Programming Help] [answered]
### Select Topic Area

Question

## Anyone interested | Кому интересно

- clone or fork this repository | клонировать этот репозиторий
- Create a bench with your own name | Создайте скамейку со своим именем
- Write your idea code about it | Напишите свой код идеи об этом
- Any idea about point **G** | Приветствуются любые идеи по поводу точки **G**.
- **In the future, this repository will become private and will be accessible to those who contribute to it || В будущем этот репозиторий станет приватным и будет доступен всем, кто в него вносит свой вклад.**

### The repo.link is https://github.com/georgetoloraia/secp-256-k1
- Mark a star so you don't miss any news | Отмечайте звездочку, чтобы не пропустить новости
- `https://github.com/georgetoloraia/secp-256-k1`

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by samus-aran** [comment, marked as answer]
Hi @georgetoloraia , we’re glad you’re here 😀

Thanks for sharing your (project/website/etc) with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
